src/cysignals/implementation.c

```diff
@@ -344,6 +364,7 @@ static void setup_alt_stack(void)
     ss.ss_size = sizeof(alt_stack);
     ss.ss_flags = 0;
     if (sigaltstack(&ss, NULL) == -1) {perror("sigaltstack"); exit(1);}
+    PLATFORM_SETUP_ALT_STACK;
```

Comment:
I don't like `PLATFORM_SETUP_ALT_STACK`.

I would rather write this as
```
#ifdef __CYGWIN__
    cygwin_setup_alt_stack();
#endif
```
just to be explicit that something is happening here only on Cygwin.

Reply:
My thinking was in theory other platforms might add something here, but I'm just as happy to do the above.

Reply:
Even if multiple platforms need to do something, I would prefer platform-specific `#ifdef`s.